packages/vite/src/node/ssr/ssrTransform.ts

```diff
@@ -31,11 +31,22 @@ export async function ssrTransform(
 ): Promise<TransformResult | null> {
   const s = new MagicString(code)
 
-  const ast = parser.parse(code, {
-    sourceType: 'module',
-    ecmaVersion: 'latest',
-    locations: true
-  }) as any
+  let ast
```

Comment:
We don't like floating variables that are not typed or at least inferred. Could you add the assumed typedef to this variable? (Seems it is `: any`?, cause of line 40, but hopefully we can find the correct type)

Reply:
that seems way outside the scope of this PR. typing it breaks a whole bunch of stuff I'm unfamiliar with. it was already typed as `any` before this PR and I'm not changing it at all

Reply:
Yeah, so you should add just `: any`

```suggestion
  let ast: any
```

Reply:
done